openassessment/xblock/static/js/openassessment-lms.min.js

```diff
@@ -1 +1 @@
ab.find(".action--submit-username").click(function(eventObject){eventObject.preventDefault(),view.loadStudentInfo()}),$manageLearnersTab.find(".action--submit-training").click(function(eventObject){eventObject.preventDefault(),view.scheduleTraining()}),$manageLearnersTab.find(".action--submit-unfinished-tasks").click(function(eventObject){eventObject.preventDefault(),view.rescheduleUnfinishedTasks()}),$staffGradeTool.find(".staff__grade__show-form").click(function(event){$(event.currentTarget).closest("."+view.baseView.SLIDABLE_CONTAINER_CLASS).hasClass(view.baseView.IS_SHOWING_CLASS)?view.closeStaffGradeForm(!1):view.loadStaffGradeForm()}))},scheduleTraining:function(){var view=this;this.server.scheduleTraining().done(function(msg){$(".schedule_training_message",view.element).text(msg)}).fail(function(errMsg){$(".schedule_training_message",view.element).text(errMsg)})},rescheduleUnfinishedTasks:function(){var view=this;this.server.rescheduleUnfinishedTasks().done(function(msg){$(".reschedule_unfinished_tasks_message",view.element).text(msg)}).fail(function(errMsg){$(".reschedule_unfinished_tasks_message",view.element).text(errMsg)})},cancelSubmission:function(submissionUUID){this.cancelSubmissionEnabled(!1);var view=this,comments=$(".cancel_submission_comments",this.element).val();this.server.cancelSubmission(submissionUUID,comments).done(function(){view.loadStudentInfo("staff-info__student__grade")}).fail(function(errorMessage){$(".cancel-submission-error").html(_.escape(errorMessage))})},cancelSubmissionEnabled:function(enabled){return this.baseView.buttonEnabled(".action--submit-cancel-submission",enabled)},comment:function(text){var $submissionComments=$(".cancel_submission_comments",this.element);if(void 0===text)return $submissionComments.val();$submissionComments.val(text)},handleCommentChanged:function(){var isBlank=""!==$.trim(this.comment());this.cancelSubmissionEnabled(isBlank)},staffSubmitEnabled:function(scope,enabled){return this.baseView.buttonEnabled(".wrapper--staff-assessment .action--submit",enabled)},assessmentRubricChanges:function(key,changesExist){changesExist&&this.baseView.unsavedWarningEnabled(!0,key,gettext("If you leave this page without submitting your staff assessment, you will lose any work you have done."))},submitStaffOverride:function(submissionID,rubric,scope){var view=this;this.callStaffAssess(submissionID,rubric,scope,function(){view.baseView.unsavedWarningEnabled(!1,view.OVERRIDE_UNSAVED_WARNING_KEY),view.loadStudentInfo("staff-info__student__grade")},".staff-override-error","regrade")},submitStaffGrade:function(submissionID,rubric,scope,continueGrading){var view=this;this.callStaffAssess(submissionID,rubric,scope,function(){view.baseView.unsavedWarningEnabled(!1,view.FULL_GRADE_UNSAVED_WARNING_KEY),view.staffGradeFormLoaded=!1,continueGrading?(view.loadStaffGradeForm(),view.baseView.scrollToTop(".openassessment__staff-area")):view.closeStaffGradeForm(!0)},".staff-grade-error","full-grade")},callStaffAssess:
```

Comment:
Let me know if you run into those odd travis failures again - I think it's a consequence of our system being brittle, not any sort of mistake on your part.

Reply:
Trying again with rebase and `make javascript`, let's see if I see the "Green" Build. 
I will update if I need you.